### DISCUSSION TITLE
label in info meaning

### DISCUSSION THREAD

**Discussion by Fan-Yixuan** [bdd100k/bdd100k#198] [Q&A]
Thank you for the data set!
I would like to ask how should I understand the labels in the 'info' annotations for videos? For example, how is the 'course' in 'locations' calculated?

**Comment by fyu** [comment]
This script should answer your question: https://doc.bdd100k.com/usage.html#trajectories

**Reply by Fan-Yixuan** [reply]
Thanks for your reply. I noticed it, but it seems to only contain the latitude and longitude information in the ‘gps’, and there is no description of the 'course' and other information that I care about.
